packages/apputils-extension/src/announcements.ts

```diff
@@ -100,7 +100,7 @@ export const announcements: JupyterFrontEndPlugin<void> = {
       if (mustFetchNews === 'none') {
         const notificationId = Notification.emit(
           trans.__(
-            'Would you like to receive official Jupyter news?\nPlease read the privacy policy.'
+            'Would you like to receive official Jupyter news?\n\nPlease read the privacy policy.'
```

Comment:
Some thoughts:
- there are numerous complaints about this popup. This change will make it even larger.
- do we need the "Please read the privacy policy" line at all? There is a link to privacy policy below anyways.
- "receive" is correct but for me it has a connotation of something more heavy weight (e.g. receive (e)mail). I feel like "get notified of" might fit here better?

```suggestion
            'Would you like to get notified official Jupyter news?'
```